gum/backend-darwin/gumkernel-darwin.c

```diff
@@ -179,14 +185,13 @@ gum_kernel_try_mprotect (gpointer address,
     return FALSE;
 
   page_size = vm_kernel_page_size;
-  aligned_address = GSIZE_TO_POINTER (
-      GPOINTER_TO_SIZE (address) & ~(page_size - 1));
+  aligned_address = address & ~(page_size - 1);
```

Comment:
Nit: `page_size` should be a `GumAddress` too, or cast to it, so the bitwise operation works on 32-bit.